package.json

```diff
@@ -97,6 +100,7 @@
     "react-player": "^2.8.2",
     "read-more-react": "^1.0.10",
     "sass": "^1.52.0",
+    "serverless-mysql": "^1.5.4",
```

Comment:
This new bundle has a price to pay increasing our bundle size: https://bundlephobia.com/package/serverless-mysql@1.5.4